### PR TITLE
Add docs for release-1.3

### PR DESCRIPTION
**Pull request by masayag**
Signed-off-by: Moti Asayag <masayag@redhat.com>

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

**Review by batzionb — APPROVED** [review]

File: docs/release-1.3/existing-rhdh.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+     echo "Retriving latest version for plugin: $PLUGIN_NAME\n";
+     curl -s -q "https://npm.registry.redhat.com/${PLUGIN_NAME}/" | jq -r '.versions | keys_unsorted[-1] as $latest_version | .[$latest_version] | "package: \"\(.name)@\(.version)\"\nintegrity: \(.dist.integrity)"';
+     echo "---"
+done
+```
+
+A sample output should look like:
+```
+Retriving latest version for plugin: @redhat/backstage-plugin-orchestrator\n
+package: "@redhat/backstage-plugin-orchestrator@1.2.0"
```
Comment (gabriel-farache) [inline docs/release-1.3/existing-rhdh.md]: do we also want to update the plugins' versions as well? They are all in 1.2

File: docs/release-1.3/existing-rhdh.md [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+        oc wait -n knative-eventing knativeeventing/knative-eventing --for=condition=Ready --timeout=5m
+        oc wait -n knative-serving knativeserving/knative-serving --for=condition=Ready --timeout=5m
+        oc wait -n openshift-serverless-logic deploy/logic-operator-rhel8-controller-manager --for=condition=Available --timeout=5m
+        oc wait -n sonataflow-infra sonataflowplatform/sonataflow-platform --for=condition=Succeed --timeout=5m
+        oc wait -n sonataflow-infra deploy/sonataflow-platform-data-index-service --for=condition=Available --timeout=5m
+        oc wait -n sonataflow-infra deploy/sonataflow-platform-jobs-service --for=condition=Available --timeout=5m
+        oc get networkpolicy -n sonataflow-infra
+        ```
+   1. Copy and execute each command from the output in your terminal. These commands ensure that all necessary services and resources in your OpenShift environment are available and running correctly.
+   1. If any service does not become available, verify the logs for that service or consult [troubleshooting steps](https://www.parodos.dev/main/docs/serverless-workflows/troubleshooting/).
```
Comment (gabriel-farache) [inline docs/release-1.3/existing-rhdh.md]: `https://www.parodos.dev/main/docs/serverless-workflows/troubleshooting` should be `https://www.parodos.dev/1.3/docs/serverless-workflows/troubleshooting`

File: docs/release-1.3/README.md [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+      oc get sonataflowclusterplatform -A
+   ```
+   If there is no cluster platform, check for a namespace-specific platform:
+   ```console
+      oc get sonataflowplatform -A
+   ```
+   Store the namespace value in SONATAFLOW_PLATFORM_NAMESPACE.
+
+4. **Set Up a Network Policy:**
+   Configure a network policy to allow traffic only between RHDH, Sonataflow services, and the workflows. The policy can be derived from [here](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/templates/network-policies.yaml)
```
Comment (gabriel-farache) [inline docs/release-1.3/README.md]: Should be `https://github.com/parodos-dev/orchestrator-helm-operator/blob/release-1.3/helm-charts/orchestrator/templates/network-policies.yaml`

File: docs/release-1.3/README.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+    In case of any failure at this stage, these jobs remain active, facilitating administrators in retrieving detailed diagnostic information to identify and address the cause of the failure.
+
+    > **Note:** that every minute on the clock a job is triggered to reconcile the CRs with the orchestrator resource values. These cronjobs are deleted when their respective features (e.g. `rhdhOperator.enabled=false`) are removed or when the orchestrator resource is removed. This is required because the CRs are not managed by helm due to the CRD dependency pre availability to the deployment of the CR.
+
+1. Apply the Orchestrator custom resource (CR) on the cluster to create an instance of RHDH and resources of OpenShift Serverless Operator and OpenShift Serverless Operator Logic.
+   Make any changes to the [CR](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/config/samples/_v1alpha1_orchestrator.yaml) before applying it, or test the default Orchestrator CR:
+    ```console
+    oc apply -n orchestrator -f https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/refs/heads/main/config/samples/_v1alpha1_orchestrator.yaml
```
Comment (gabriel-farache) [inline docs/release-1.3/README.md]: Should be `https://github.com/parodos-dev/orchestrator-helm-operator/blob/release-1.3/config/samples/_v1alpha1_orchestrator.yaml`
and 
`https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/refs/heads/release-1.3/config/samples/_v1alpha1_orchestrator.yaml`

File: docs/release-1.3/README.md [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+      installPlanApproval: Automatic
+      name: orchestrator-operator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+    ```
+
+1.  Run the following commands to determine when the installation is completed:
+
+    ```console
+    wget https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/main/hack/wait_for_operator_installed.sh -O /tmp/wait_for_operator_installed.sh && chmod u+x /tmp/wait_for_operator_installed.sh && /tmp/wait_for_operator_installed.sh
```
Comment (gabriel-farache) [inline docs/release-1.3/README.md]: Should be `https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/release-1.3/hack/wait_for_operator_installed.sh`

File: docs/release-1.3/README.md [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+1. Create a namespace for the Red Hat Developer Hub Operator (RHDH Operator):
+
+   ```console
+   oc new-project rhdh-operator
+   ```
+
+1.  Download the setup script from the github repository and run it to create the RHDH secret and label the GitOps namespaces:
+
+    ```console
+    wget https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/main/hack/setup.sh -O /tmp/setup.sh && chmod u+x /tmp/setup.sh
```
Comment (gabriel-farache) [inline docs/release-1.3/README.md]: Should be `https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-operator/release-1.3/hack/setup.sh`

File: docs/release-1.3/README.md [hunk 7]
```diff
@@ -0,0 +1,309 @@
...
+        oc wait -n knative-eventing knativeeventing/knative-eventing --for=condition=Ready --timeout=5m
+        oc wait -n knative-serving knativeserving/knative-serving --for=condition=Ready --timeout=5m
+        oc wait -n openshift-serverless-logic deploy/logic-operator-rhel8-controller-manager --for=condition=Available --timeout=5m
+        oc wait -n sonataflow-infra sonataflowplatform/sonataflow-platform --for=condition=Succeed --timeout=5m
+        oc wait -n sonataflow-infra deploy/sonataflow-platform-data-index-service --for=condition=Available --timeout=5m
+        oc wait -n sonataflow-infra deploy/sonataflow-platform-jobs-service --for=condition=Available --timeout=5m
+        oc get networkpolicy -n sonataflow-infra
+        ```
+   1. Copy and execute each command from the output in your terminal. These commands ensure that all necessary services and resources in your OpenShift environment are available and running correctly.
+   1. If any service does not become available, verify the logs for that service or consult [troubleshooting steps](https://www.parodos.dev/main/docs/serverless-workflows/troubleshooting/).
```
Comment (gabriel-farache) [inline docs/release-1.3/README.md]: Should be `https://www.parodos.dev/1.3/docs/serverless-workflows/troubleshooting/`

File: docs/release-1.3/README.md [hunk 8]
```diff
@@ -0,0 +1,309 @@
...
+
+  Remember to enable [argocd](https://github.com/parodos-dev/orchestrator-helm-operator/blob/c577e95e063e2bf8119b2b23890df04792f9424c/config/crd/bases/rhdh.redhat.com_orchestrators.yaml#L451) and [tekton](https://github.com/parodos-dev/orchestrator-helm-operator/blob/c577e95e063e2bf8119b2b23890df04792f9424c/config/crd/bases/rhdh.redhat.com_orchestrators.yaml#L443) in your CR instance.
+
+## Detailed Installation Guide
+### From OperatorHub
+1. Deploying PostgreSQL reference implementation
+   - **If you do not have a PostgreSQL instance in your cluster** \
+   you can deploy the PostgreSQL reference implementation by following the steps here.
+   - **If you already have PostgreSQL running in your cluster** \
+   ensure that the default settings in the [PostgreSQL values](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/postgresql/values.yaml) file match those provided in the [Orchestrator values](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/values.yaml) file.
```
Comment (gabriel-farache) [inline docs/release-1.3/README.md]: Should be `https://github.com/parodos-dev/orchestrator-helm-operator/blob/release-1.3/helm-charts/orchestrator/values.yaml`

**Review by gabriel-farache — CHANGES_REQUESTED** [review]
Few links are still pointing to `main` branches instead of 1.3/release-1.3 (even if they are not yet created)

File: docs/release-1.3/existing-rhdh.md (hunk 1 above)
Comment (masayag) [reply docs/release-1.3/existing-rhdh.md]: This is the sample, I'll update it to 1.3 once the 1.3 GA plugins are available. I'll create an issue in the meantime not to miss this.

**Comment by masayag** [comment]
Thank you @gabriel-farache for a thorough review!

**Comment by masayag** [comment]
@jordigilh for this PR to get merged, we need to create a `release-1.3` branch and start using it for the release instead of the main. 
We should do that before we merge https://github.com/parodos-dev/orchestrator-helm-operator/pull/247

**Review by gabriel-farache — APPROVED** [review]

File: docs/release-1.3/existing-rhdh.md (hunk 1 above)
Comment (masayag) [reply docs/release-1.3/existing-rhdh.md]: updated to 1.3.0 plugin version with their integrity values.

**Pull request merged**